### ISSUE TITLE
Callset call_method field incorrectly formatted

### ISSUE THREAD

**Issue by kellrott** [bmeg/bmeg-etl#215]
The current `CallSet.call_method` is incorrectly filled in for MC3 data. It should be on method per callset, but right now its a concatenation of all the methods that made common calls, leading to a weird number of callsets. There should be a separate callset for each of the methods split by the `|` and the `*` character should be removed from the names. The the callset with call_method `RADIA|MUTECT|MUSE|VARSCANS` should be 4 different entries. 

Once data is correctly formatted the values found in `call_method` should only be one the values in `["RADIA", "MUTECT", "MUSE", "VARSCANS", "INDELOCATOR", "VARSCANI", "PINDEL", "SOMATICSNIPER"]`

The current formatting contains various permutations of combinations:

```
[
    {
        "gid": "Callset:mc3:d414b3fe-b768-4a98-b285-5284bffa66f9:8404ab78-054d-4cd1-b92e-951f3047ff45:MUTECT|MUSE",
        "label": "Callset",
        "data": {
            "call_method": "MUTECT|MUSE",
            "normal_aliquot_id": "8404ab78-054d-4cd1-b92e-951f3047ff45",
            "source": "mc3",
            "tumor_aliquot_id": "d414b3fe-b768-4a98-b285-5284bffa66f9"
        }
    },
    {
        "gid": "Callset:mc3:d414b3fe-b768-4a98-b285-5284bffa66f9:8404ab78-054d-4cd1-b92e-951f3047ff45:MUSE|VARSCANS",
        "label": "Callset",
        "data": {
            "call_method": "MUSE|VARSCANS",
            "normal_aliquot_id": "8404ab78-054d-4cd1-b92e-951f3047ff45",
            "source": "mc3",
            "tumor_aliquot_id": "d414b3fe-b768-4a98-b285-5284bffa66f9"
        }
    },
    {
        "gid": "Callset:mc3:d414b3fe-b768-4a98-b285-5284bffa66f9:8404ab78-054d-4cd1-b92e-951f3047ff45:RADIA|MUTECT|MUSE|VARSCANS",
        "label": "Callset",
        "data": {
            "call_method": "RADIA|MUTECT|MUSE|VARSCANS",
            "normal_aliquot_id": "8404ab78-054d-4cd1-b92e-951f3047ff45",
            "source": "mc3",
            "tumor_aliquot_id": "d414b3fe-b768-4a98-b285-5284bffa66f9"
        }
    },
    {
        "gid": "Callset:mc3:d414b3fe-b768-4a98-b285-5284bffa66f9:8404ab78-054d-4cd1-b92e-951f3047ff45:INDELOCATOR*|VARSCANI*|PINDEL",
        "label": "Callset",
        "data": {
            "call_method": "INDELOCATOR*|VARSCANI*|PINDEL",
            "normal_aliquot_id": "8404ab78-054d-4cd1-b92e-951f3047ff45",
            "source": "mc3",
            "tumor_aliquot_id": "d414b3fe-b768-4a98-b285-5284bffa66f9"
        }
    },
    {
        "gid": "Callset:mc3:d414b3fe-b768-4a98-b285-5284bffa66f9:8404ab78-054d-4cd1-b92e-951f3047ff45:RADIA|MUTECT|MUSE",
        "label": "Callset",
        "data": {
            "call_method": "RADIA|MUTECT|MUSE",
            "normal_aliquot_id": "8404ab78-054d-4cd1-b92e-951f3047ff45",
            "source": "mc3",
            "tumor_aliquot_id": "d414b3fe-b768-4a98-b285-5284bffa66f9"
        }
    },
    {
        "gid": "Callset:mc3:d414b3fe-b768-4a98-b285-5284bffa66f9:8404ab78-054d-4cd1-b92e-951f3047ff45:MUTECT|MUSE|VARSCANS",
        "label": "Callset",
        "data": {
            "call_method": "MUTECT|MUSE|VARSCANS",
            "normal_aliquot_id": "8404ab78-054d-4cd1-b92e-951f3047ff45",
            "source": "mc3",
            "tumor_aliquot_id": "d414b3fe-b768-4a98-b285-5284bffa66f9"
        }
    },
    {
        "gid": "Callset:mc3:d414b3fe-b768-4a98-b285-5284bffa66f9:8404ab78-054d-4cd1-b92e-951f3047ff45:MUTECT|RADIA|SOMATICSNIPER|MUSE|VARSCANS",
        "label": "Callset",
        "data": {
            "call_method": "MUTECT|RADIA|SOMATICSNIPER|MUSE|VARSCANS",
            "normal_aliquot_id": "8404ab78-054d-4cd1-b92e-951f3047ff45",
            "source": "mc3",
            "tumor_aliquot_id": "d414b3fe-b768-4a98-b285-5284bffa66f9"
        }
    },
    {
        "gid": "Callset:mc3:d414b3fe-b768-4a98-b285-5284bffa66f9:8404ab78-054d-4cd1-b92e-951f3047ff45:MUTECT|SOMATICSNIPER|MUSE",
        "label": "Callset",
        "data": {
            "call_method": "MUTECT|SOMATICSNIPER|MUSE",
            "normal_aliquot_id": "8404ab78-054d-4cd1-b92e-951f3047ff45",
            "source": "mc3",
            "tumor_aliquot_id": "d414b3fe-b768-4a98-b285-5284bffa66f9"
        }
    },
    {
        "gid": "Callset:mc3:d414b3fe-b768-4a98-b285-5284bffa66f9:8404ab78-054d-4cd1-b92e-951f3047ff45:RADIA|SOMATICSNIPER|VARSCANS",
        "label": "Callset",
        "data": {
            "call_method": "RADIA|SOMATICSNIPER|VARSCANS",
            "normal_aliquot_id": "8404ab78-054d-4cd1-b92e-951f3047ff45",
            "source": "mc3",
            "tumor_aliquot_id": "d414b3fe-b768-4a98-b285-5284bffa66f9"
        }
    },
    {
        "gid": "Callset:mc3:d414b3fe-b768-4a98-b285-5284bffa66f9:8404ab78-054d-4cd1-b92e-951f3047ff45:RADIA|VARSCANS",
        "label": "Callset",
        "data": {
            "call_method": "RADIA|VARSCANS",
            "normal_aliquot_id": "8404ab78-054d-4cd1-b92e-951f3047ff45",
            "source": "mc3",
            "tumor_aliquot_id": "d414b3fe-b768-4a98-b285-5284bffa66f9"
        }
    }
]
```

When the data is correctly formatted the following query should work. 
```
q = O.query().V().where(gripql.eq("_label", "Individual")).where(gripql.eq("gdc_attributes.project.project_id", "TCGA-BRCA"))
q = q.in_("BiosampleFor").in_("AliquotFor").in_("CallsetFor")
q = q.where(aql.eq("call_method", "MUTECT"))
```

**Comment by bwalsh** [comment]
@kellrott : can you close if all ok?

**Comment by bwalsh** [comment]
@kellrott : you were going to revert this. `s/call_method/call_methods/`

**Comment by kellrott** [comment]
Because the variant calls were already merged via a meta-caller pipeline, How about, one Callset for the metacaller run (per patient), rather then 7 callsets for all the methods. Then store the methods on the VariantCall edge:

So on the `Callset` Vertex remove `call_method` Then on the VariantCall edge add `call_method` field as an array
```
  call_method: ["MUTECT", "SOMATICSNIPER", "MUSE" ]
```

**Comment by bwalsh** [comment]
@kellrott 

* addressed in #223 
* **`edge size reduced from 11GB to  4676 MB`**

### old

```
         relation          | kind |  size
---------------------------+------+---------
 public.edge               | r    | 11 GB
 public.vertex_data        | i    | 5399 MB
 public.edge_gid           | i    | 3978 MB
 public.edge_label_to_from | i    | 3741 MB
 public.edge_label_from_to | i    | 3732 MB
 public.vertex             | r    | 1666 MB
 public.vertex_gid         | i    | 351 MB
 public.vertex_label       | i    | 129 MB
```

### new
```

         relation          | kind |  size
---------------------------+------+---------
 public.vertex_data        | i    | 5399 MB
 public.edge               | r    | 4676 MB
 public.edge_gid           | i    | 1686 MB
 public.vertex             | r    | 1645 MB
 public.edge_label_to_from | i    | 1581 MB
 public.edge_label_from_to | i    | 1579 MB
 public.vertex_gid         | i    | 343 MB
 public.vertex_label       | i    | 128 MB
```
